kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/ListCustomResourceDefinitions.java

```diff
@@ -28,19 +28,20 @@
 public class ListCustomResourceDefinitions {
   public static void main(String[] args) {
     try {
```

Comment:
Please reuse this try-catch block to init your resources

Reply:
(same for the rest)

Reply:
@manusa Does this look good now or something else was expected?

Reply:
Just asking so that I can make similar changes to the rest of the files

Reply:
Yes, the point is to keep a single try-catch-finally block and reduce complexity.
In these examples it doesn't make any sense to add those nested blocks.

Please also squash your commits once your changes are done, thx.

Reply:
@manusa Did the required changes and squashed the commits